### ISSUE TITLE
account created using create_account_delegated is unable to operate

### ISSUE THREAD

**Issue by gropox** [GolosChain/golos#155]
account created using create_account_delegated is unable to operate. For example such account cannot post, vote etc.

Probably caused by zero vesting shares. At first, on steemit it is impossible to create an account using delegation with zero vesting shares. If you pass as fee "0.000 STEEM", steem complains about min steem amount (actually 0.200 STEEM as fee and around 60k VESTS delegation are required). 

```
0 exception: unspecified
10 assert_exception: Assert Exception
o.fee >= wso.median_props.account_creation_fee: Insufficient Fee: 0.200 STEEM required, 0.000 STEEM provided.
    {"f":"0.200 STEEM","p":"0.000 STEEM"}
    th_a  steem_evaluator.cpp:226 do_apply
```

```
10 assert_exception: Assert Exception
current_delegation >= target_delegation: Inssufficient Delegation 61986.259960 VESTS required, 12000.000000 VESTS provided.
    {"f":"61986.259960 VESTS","p":"12000.000000 VESTS","account_creation_fee":"0.200 STEEM","o.fee":"0.000 STEEM","o.delegation":"12000.000000 VESTS"}
    th_a  steem_evaluator.cpp:222 do_apply
```

```
create_account_delegated ropox "0.200 STEEM" "61986.259960 VESTS" gropox "{}" true
unlocked >>> get_account gropox

  "reward_vesting_balance": "0.000000 VESTS",
  "reward_vesting_steem": "0.000 STEEM",
  "vesting_shares": "413.241667 VESTS",
  "delegated_vesting_shares": "0.000000 VESTS",
  "received_vesting_shares": "61986.259960 VESTS",
  "vesting_withdraw_rate": "0.000000 VESTS",

unlocked >>> post_comment gropox "permlink1" "" "test" "title1" "body1" "{}" true

```

On golosd you can pass zero amounts. Second, despite non zero fees, the new account has zero vesting shares. and is unable to post. 

```
create_account_delegated cyberfounder "10.200 GOLOS" "72000.000000 GESTS" gropox2 "{}" true
unlocked >>> get_account gropox2

  "vesting_shares": "0.000000 GESTS",
  "delegated_vesting_shares": "0.000000 GESTS",
  "received_vesting_shares": "72000.000000 GESTS",
  "vesting_withdraw_rate": "0.000000 GESTS",

unlocked >>> post_comment gropox2 "permlink1" "" "test" "title1" "body1" "{}" true
0 exception: unspecified
10 assert_exception: Assert Exception
has_bandwidth: Account exceeded maximum allowed bandwidth per vesting share.
    {"account_vshares":"0","account_average_bandwidth":"119000000","max_virtual_bandwidth":"4137970271926965589","total_vesting_shares":"355322585124"}
    th_a  database.cpp:722 update_account_bandwidth

```

**Comment by gropox** [comment]
after transferring to vesting, the account created using delegation is operational.

**Comment by Nemo1369** [comment]
Fix proposed with c4beb5428a6fba502eba8ad6242c4cbbc7e9d5ea. The trouble was a redundant fee assignment in a create_account_with_keys_delegated.

**Comment by Nemo1369** [comment]
Fixed with c4beb54
